docs/root/version_history/current.rst

```diff
@@ -24,7 +24,7 @@ Bug Fixes
 ---------
 *Changes expected to improve the state of the world and are unlikely to have negative effects*
 
-* dns: fix a bug where custom resolvers provided in configuration were not preserved after network issues.
+* dns_filter: correctly associate DNS response IDs when multiple queries are received.
```

Comment:
merge issue?

Reply:
I know there was a conflict earlier that I thought I resolved.   The entry should be "dns_filter" since that's the tag I had used in prior commits.